scripts/start.sh

```diff
@@ -122,6 +122,9 @@ if [ -n "${SERVER_PASSWORD}" ]; then
     SERVER_PASSWORD=$(sed -e 's/^"\(.*\)"$/\1/' -e "s/^'\(.*\)'$/\1/" <<< "$SERVER_PASSWORD")
     echo "SERVER_PASSWORD=${SERVER_PASSWORD}"
     sed -E -i "s/ServerPassword=\"[^\"]*\"/ServerPassword=\"$SERVER_PASSWORD\"/" /palworld/Pal/Saved/Config/LinuxServer/PalWorldSettings.ini
+else
+    echo "No SERVER_PASSWORD"
+    sed -E -i "s/ServerPassword=\"[^\"]*\"/ServerPassword=\"\"/" /palworld/Pal/Saved/Config/LinuxServer/PalWorldSettings.ini
```

Comment:
```suggestion
elif [ -v SERVER_PASSWORD ]; then
    echo "No SERVER_PASSWORD"
    sed -E -i "s/ServerPassword=\"[^\"]*\"/ServerPassword=\"\"/" /palworld/Pal/Saved/Config/LinuxServer/PalWorldSettings.ini
```

I think we should remove SERVER_PASSWORD from the Dockerfile. 
If by default we have no variable named SERVER_PASSWORD then nothing will be overridden unless you explicitly state no password (With the elif -v). 

This way we don't always overwrite the password. I'm not worried about the extra write so much as removing a password someone has set in their file but not set as an environmental variable.

Reply:
I also think this way is better. Thanks for your suggestion.